### PR TITLE
Add the attribute links to the device data

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
The reason I need this is for the webui to be able to compare the paths that cockpit returns for the mount point selection with the devices that anaconda knows about.
I have noticed that for LVM cockpit returns something like:
"/dev/vgroup0/lvol0"

Whilst anaconda in the path property in device data has:
"/dev/mapper/vgroup0-lvol0"

All these are symlinks, so I thought its sane to do the discovery if the two paths are the same in the front end. CC @vojtechtrefny

File: pyanaconda/modules/common/structures/storage.py [hunk 1]
```diff
@@ -140,6 +141,18 @@ def children(self) -> List[Str]:
     def children(self, value):
         self._children = value
 
+    @property
+    def links(self) -> List[Str]:
+        """Children of the device.
+
+        :return: a list of device paths
```
Comment (VladimirSlavik) [inline pyanaconda/modules/common/structures/storage.py:148]: Why is it called "links" if it's "children_paths" ?

Comment (jkonecny12) [reply pyanaconda/modules/common/structures/storage.py:148]: I think it's because these are actually symlinks to the devices?

Comment (KKoukiou) [reply pyanaconda/modules/common/structures/storage.py:148]: Sorry I need to update the docstring, it's not children, it's symlinks indeed.

**Review by M4rtinK — APPROVED** [review]
Looks correct Python code wise, but I'm afraid I lack the necessary context to spot possible deeper issues. :P

**Review by vojtechtrefny — COMMENTED** [review]
This is ok as a temporary solution, but we'll need to change this in the future for btrfs (where device path itself isn't enough information).

**Comment by KKoukiou** [comment]
> This is ok as a temporary solution, but we'll need to change this in the future for btrfs (where device path itself isn't enough information).

Not merging this yet then, will check if I can find some more global solution.

**Pull request merged**